schemes/musica/musica_ccpp.meta

```diff
@@ -19,6 +19,12 @@
   type = integer
   dimensions = ()
   intent = in
+[ photolysis_wavelength_grid_interfaces ]
+  standard_name = photolysis_wavelength_grid_interfaces
+  units = m
```

Comment:
In the configuration (`ts1_tsmlt.json`), the unit is `nm`. Are we going to use `m` and to add conversion?

Reply:
I saw the conversion. never mind!